gitstrap.go

```diff
@@ -340,6 +320,19 @@ func (strap *strapCtx) addCollaborators(owner string, repo *github.Repository) e
 	return nil
 }
 
+func getOwner(strap *strapCtx, opt Options) (string, error) {
```

Comment:
@vassilevsky can we add a unit test for new function?

Reply:
I have added the first minimal test. Took me a while to explore Go testing ecosystem :) Will continue extending the test to cover all cases.

Reply:
I have covered all cases.